<?xml version="1.0"?>
<launch>

<arg name="baudrate" 	  default="115200" />
<arg name="device" 		  default="/dev/ttyACM0" />
<arg name="image_height"  default="128" />
<arg name="image_width"   default="128" />
<arg name="image_decoder" default="jpeg" />
<arg name="image_mode" 	  default="L" />
<arg name="image_topic"   default="/image_raw" />

<arg name="use_compressed" default="true" />  

<node pkg="waypoint_navigation" name="openmv_node" type="openmv_node.py" output="screen"> 
	<param name="baudrate" 		value="115200"/> 
	<param name="device" 		value="/dev/ttyACM0"/> 

	<param name="image_height" 	value="$(arg image_height)"/> 
	<param name="image_width" 	value="$(arg image_width)"/> 
	<param name="image_decoder" value="$(arg image_decoder)"/> 
	<param name="image_mode" 	value="$(arg image_mode)"/> 
	<param name="image_topic" 	value="$(arg image_topic)"/> 


	<param name="use_compressed" value="$(arg use_compressed)" /> 

</node>

</launch>
